src/plugins/yawn/README.md

```diff
@@ -0,0 +1,74 @@
+- infos = Information about the yawn plugin is in keys below
+- infos/author = René Schwaiger <sanssecours@me.com>
+- infos/licence = BSD
+- infos/needs = directoryvalue yamlsmith
+- infos/provides = storage/yaml
+- infos/recommends =
+- infos/placements = getstorage
+- infos/status = maintained unittest preview memleak experimental unfinished nodoc concept discouraged
+- infos/metadata =
+- infos/description = This storage plugin use a Early parser to read YAML files
+
+# YAwn
+
+## Introduction
+
+This plugin uses [YAEP](https://github.com/vnmakarov/yaep) to  parse the [YAML](http://yaml.org) serialization format.
```

Comment:
I would simply say "to  parse [YAML](http://yaml.org)." We did not define what a "serialization format" is and how this differs from a configuration file format."